open-ce/docker_build.py

```diff
@@ -80,7 +82,12 @@ def _add_volume(local_path, container_path):
 
     return volume_arg
 
-def _create_container(container_name, image_name, output_folder):
+def _mount_name(folder_path):
+    if len(folder_path) <= 1:
+        return ""
+    return _mount_name(os.path.dirname(folder_path)) + os.path.basename(folder_path)
+
+def _create_container(container_name, image_name, output_folder, env_folders):
```

Comment:
@bnemanich 
I'm using this function to generate a name for each env file mount point.